tests/benchdnn/eltwise/eltwise.cpp

```diff
@@ -78,6 +78,8 @@ static int compare(const prb_t *p, const dnn_mem_t &mem_fp,
         trh = 4e-6;            // subtract, which leads to low accuracy.
     if (p->alg == alg_t::ELU) // when x -> -0, a(exp(-x) - 1) has cancellation
         trh = 4e-5;           // subtract, which leads to low accuracy.
+    if (p->alg == alg_t::SWISH) 
+        trh = 4e-5;           
```

Comment:
The only problem I noticed was in UT test_eltwise , to broad range for input filler could result in
failure. So I followed ELU settings since then, but I just checked that benchDNN eltwise works fine(my AVX2 platform) would this relaxed threshold so It will be reverted.